Alignment/OfflineValidation/scripts/haddws.C

```diff
@@ -0,0 +1,304 @@
+/******************************************************************************
+ * This is evolved from the original macro $ROOTSYS/tutorial/io/hadd.C 
+ * which is provided as a tutorial with the ROOT package.
+ * http://root.cern.ch/drupal/content/downloading-root
+ *
+ * I have added several very useful features to that:
+ * 1. Merge/Add histograms with weights.
+ *		For this, a weight should be specified for each of the input root files.
+ *		Remark: Unfortunately there is no easy way to merger trees with weight
+ *		         so they are just added together (the default behaviour of ROOT)
+ *		         ignoring the specified weights.
+ * 2. Command line execution. No need to load ROOT interpreter every time.
+ *
+ * @Author  : Samantha Hewamanage <samanthaATSPAMMENOTfnal.gov>
+ * @Employer: Florida International University for the CMS Colloboration
```

Comment:
I am not entirely sure what's the CMS policy for shamelessy stealing open source code (probably the cmssw license file should tell something about it), but at the very list I think the original repo should be referenced: https://github.com/hkaushalya/haddws  (by the way it's under GPL 3.0).